app/models/employer_profile.rb

```diff
@@ -1102,7 +1104,7 @@ def is_attestation_eligible?
 
   def validate_and_send_denial_notice
```

Comment:
Please remove this trigger ie. after_save call back on organization model. There is a possibility that the notice might trigger multiple times. Lets us only depend on plan year state changes to trigger the notice